code/modules/transport/pods/preassembled_frame_kit.dm

```diff
@@ -0,0 +1,270 @@
+			if (isscrewingtool(W))
+				user.visible_message("[user] begins screwing the pod's maintenance panels shut...")
+				playsound(src.loc, 'sound/items/Ratchet.ogg', 50, 1)
+				SETUP_GENERIC_ACTIONBAR(user, src, src.step_build_time, /obj/structure/preassembeled_vehicleframe/proc/step_screw_2, list(user), \
+			 		W.icon, W.icon_state, "With the cockpit and exterior indicators secured, the control system automatically starts up.", null)
+			else
+				boutput(user, "You need a screwdriver to close the maintenance panels.")
+
+/obj/structure/preassembeled_vehicleframe/proc/step_wrench_1(var/mob/user)
+	src.overlays += image(src.icon, "[pick("frame1", "frame2")]")
+	src.stage = BUILD_STEP_WRENCH_1
+	src.help_message = "Use a welder to weld the joints together."
+
+/obj/structure/preassembeled_vehicleframe/proc/step_weld_1(var/mob/user)
+	src.overlays -= image(src.icon, "frame1")
+	src.overlays -= image(src.icon, "frame2")
+	src.icon_state = "frame"
+	src.stage = BUILD_STEP_WELD_1
+	src.help_message = "Use a screwdriver to screw the circuit boards and the engine together."
+
+/obj/structure/preassembeled_vehicleframe/proc/step_screw_1(var/mob/user)
+	src.overlays += image(src.icon, "wires")
+	src.overlays += image(src.icon, "circuits")
+	src.stage = BUILD_STEP_SCREW_1
+	src.help_message = "Use any kind of pod armor from a manufacturer."
+
+/obj/structure/preassembeled_vehicleframe/proc/step_armor(var/mob/user, var/obj/item/podarmor/armor)
+	user.u_equip(armor)
+	qdel(armor)
+	src.overlays += image(src.icon, armor.overlay_state)
+	src.stage = BUILD_STEP_ARMOR
+	src.help_message = "Use a wrench to secure the pod's thrusters and control system."
+	src.armor_type = armor.type
+	src.vehicle_type = armor.vehicle_types["[src.type]"]
+	if(istype(armor, /obj/item/podarmor/armor_custom))
+		src.setMaterial(armor.material)
+
+/obj/structure/preassembeled_vehicleframe/proc/step_wrench_2(var/mob/user)
+	src.overlays += image(src.icon, "thrust")
+	src.overlays += image(src.icon, "control")
+	src.stage = BUILD_STEP_WRENCH_2
+	src.help_message = "Use a welder to weld the exterior."
+
+/obj/structure/preassembeled_vehicleframe/proc/step_weld_2(var/mob/user)
+	src.overlays += image(src.icon, "covers")
+	src.stage = BUILD_STEP_WELD_2
+	src.help_message = "Use a screwdriver to close the maintenance panels."
+
+/obj/structure/preassembeled_vehicleframe/proc/step_screw_2(var/mob/user)
+	var/obj/machinery/vehicle/V = new vehicle_type( src.loc )
+	if (src.armor_type == /obj/item/podarmor/armor_custom)
+		V.name = src.vehicle_name
+		V.setMaterial(src.material)
+	logTheThing(LOG_STATION, user, "finishes building a [V] in [get_area(user)] ([log_loc(user)])")
+	qdel(src)
+
+/*-----------------------------*/
+/* Deconstruction              */
+/*-----------------------------*/
+
+/obj/structure/preassembeled_vehicleframe/verb/deconstruct()
+	set src in oview(1)
+	set category = "Local"
+
+	if (usr.stat)
+		return
+
+	boutput(usr, "Deconstructing frame...")
```

Comment:
I'd prefer a visible_message informing others of the action being performed than boutput.